regexes/client/browsers.yml

```diff
@@ -5,6 +5,13 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# BriskBard (https://www.briskbard.com/)
+- regex: 'BriskBard(?:/(\d+[\.\d]+))?'
+  name: 'BriskBard'
+  version: '$1'
+  engine:
+    default: 'Blink'
```

Comment:
According to the information from the official website, browser has two engines: Blink,Trident 

```yml
- regex: Chrome.+BriskBard/(\d+[\.\d]+)
  name: 'BriskBard'
  version: '$1'
  engine:
    default: 'Blink'
- regex: 'BriskBard(?:/(\d+[\.\d]+))?'
  name: 'BriskBard'
  version: '$1'
```